src/codecs/avif/encoder-meta.ts

```diff
@@ -12,12 +14,14 @@ export const label = 'AVIF';
 export const mimeType = 'image/avif';
 export const extension = 'avif';
 export const defaultOptions: EncodeOptions = {
-  minQuantizer: 16,
-  maxQuantizer: 16,
+  minQuantizer: 33,
+  maxQuantizer: 63,
+  minQuantizerAlpha: 33,
+  maxQuantizerAlpha: 63,
   tileColsLog2: 0,
   tileRowsLog2: 0,
-  speed: 10,
-  subsample: 0,
+  speed: 8,
```

Comment:
This is the default in the reference encoder and the speed seems reasonable. 7 brings significant improvements but is significantly slower.